src/Avalonia.Base/Media/MediaContext.cs

```diff
@@ -16,7 +16,7 @@ internal partial class MediaContext : ICompositorScheduler
     private TimeSpan _inputMarkerAddedAt;
     private bool _isRendering;
     private bool _animationsAreWaitingForComposition;
-    private const double MaxSecondsWithoutInput = 1;
+    private const double MaxSecondsWithoutInput = 0.2;
```

Comment:
We should probably make it configurable, otherwise we'll risk running into issues with XPF too.

Reply:
Any preference as to where this configuration should be placed?

Reply:
We probably need a separate options class. DispatcherOptions, maybe?

We should also consider runtimeconfig.json-based settings at some point.

Reply:
Done. Please check that the naming and doc comments are to your liking.